### ISSUE TITLE
OWA: Inline attachment image rendered on receiver

### ISSUE THREAD

**Issue by sjoerd222888** [OfficeDev/office-js#3025]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

## Your Environment

* Platform: Office on the web
* Host: Outlook
* Operating System: MacOS 13.0.1
* Browser: Chrome Version 108.0.5359.94 (Official Build) (arm64)
 
## Expected behavior

Image in message for meeting description is rendered only once and not listed as separate attachment again. Same as the sender sees the message.

![image](https://user-images.githubusercontent.com/10337756/206191691-2b5df51e-2f90-4cf8-a899-3d9a3bdd3c0d.png)

## Current behavior

For the invitee image in message is rendered but also listed separate attachment.

![image](https://user-images.githubusercontent.com/10337756/206191649-b6f13ca1-c81d-4b67-b7bf-114ca8250d7f.png)

## Steps to reproduce

Create an outlook calendar event an in the add an image like this:
```
Office.context.mailbox.item.addFileAttachmentFromBase64Async(logoBase64, "square.png", { isInline: true }, (attachResult) => {
    if (attachResult.status === Office.AsyncResultStatus.Succeeded) {
        resolve("square.png") // attachment-id to be used for img source, see below
    } else {
        reject(
            'Error setting image attachment. ' +
            attachResult.error.message
        );
    }
    });
```

And use it a meeting description like this:
```
<img id="square" src="cid:square.png" style='width:90px' alt='Square'>
```

# Provide additional details

When adding the image manually by hand and not by code what outlook seems to render everything correctly. So it seems when using `addFileAttachmentFromBase64Async` together with the option `{ isInline: true }` the attachment is nevertheless not considered inline. The eml file of the invite receiver shows the following:

```
Content-Disposition: attachment; filename="square.png"; size=864;
```

The message body is create like this:
```
Office.context.mailbox.item.addFileAttachmentFromBase64Async(logoBase64, "square.png", { isInline: true }, (attachResult) => {
    if (attachResult.status === Office.AsyncResultStatus.Succeeded) {
       const body =  `\
                    <div>
                        <img id="square" src="cid:square.png" style='width:90px' alt='Square'>
                        <br>
                        <span>Test image attachment</span>
                    </div>
                `
       Office.context.mailbox.item.body.setAsync(
         body,
         {
               coercionType: Office.CoercionType.Html,
          },
          () => resolve(null)
       );
    } else {
        reject(
            'Error setting image attachment. ' +
            attachResult.error.message
        );
    }
});
```
## Context

We develop an outlook add-in and having the image rendered twice, while functionally the rest works, is considered a UI glitch which prevents to ship the add-in.

## Useful logs
(none)

**Comment by exextoc** [comment]
Thanks for reporting this issue regarding the event attachment. It has been put on our backlog. We, unfortunately, have no timelines to share at this point 

Internal tracking id: Office: 183411
